### PR TITLE
Us21

### LINKED ISSUE #21: Admin Dashboard Links
As an admin,
When I visit the admin dashboard (/admin)
Then I see a link to the admin merchants index (/admin/merchants)
And I see a link to the admin invoices index (/admin/invoices)

### PR DESCRIPTION
**Pull request by gregoryjflaherty**
## Description of Changes
closes #21 

* Changed scope to namespaces- only way path via routes would work
* Added links for admin dashboard to link to admin/merchants & admin/invoices
* All tests passing 

## Concerns (tags recommended)

* No current concerns - wondering why namespaces would work but scope wouldn't 

## What gif describes how you feel about this PR?

fresh_prince_i_dont_know

### CODE REVIEW

File: spec/features/admin/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    it 'has links to invoices index' do
+      visit '/admin/'
+      expect(page).to have_link("All Invoices")
+
+      click_on "All Invoices"
+      expect(current_path).to eq('/admin/invoices')
+    end
+  end
+  #   As an admin,
```
Comment (glsorensen) [inline spec/features/admin/index_spec.rb]: I think that you can remove lines 29+

**Review by glsorensen — CHANGES_REQUESTED** [review]

File: spec/features/admin/index_spec.rb (hunk 1 above)
Comment (gregoryjflaherty) [reply spec/features/admin/index_spec.rb]: Perfect. Done

File: config/routes.rb [hunk 2]
```diff
@@ -3,9 +3,16 @@
...
-    
+
     resources :items, controller: 'merchant_items', except: [:destroy]
 
     resources :dashboard, controller: 'merchant_dashboard', only: [:index]
   end
+
+  resources :admin, only: [:index]
+
+  namespace :admin do
```
Comment (jgrazulis) [inline config/routes.rb:14]: interesting I wanna play around with this a bit when I pull to see what we can do with the other resources/controller syntax

File: app/controllers/admin/invoices_controller.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Admin::InvoicesController < AdminController
```
Comment (jgrazulis) [inline app/controllers/admin/invoices_controller.rb:1]: I am wondering if it's a good idea to continue creating files/methods we don't yet need. I get being prepared, but looking at all the methods created in some of the other controllers I don't think it was necessary.

**Review by glsorensen — APPROVED** [review]

**Pull request merged**